src/kpi_app/app.py

```diff
@@ -439,8 +433,8 @@ def create_card(title, content1, content2):
     return card
 
 
-def create_card_Autorinnen(edits=10, interest="all", subject="all"):
-    var_1 = Anzahl_Autorinnen(
+def create_card_authors(edits=10, interest="all", subject="all"):
+    var_1 = get_number_of_authors(
```

Comment:
Not only here but `var_1` etc. is not a good name for a variable, we should always try to find a descriptive variable name.